content/deployment/quickstart.md

```diff
@@ -5,23 +5,66 @@ chapter: false
+
+### Verifying our release
+
+{{% notice note %}}
+Releases are signed using [use our GPG key](https://coreruleset.org/security.asc), (fingerprint: 3600 6F0E 0BA1 6783 2158 8211 38EE ACA1 AB8A 6E72). You can verify the release using GPG/PGP compatible tooling.
+
+To get our key using gpg from public servers: `gpg --keyserver pgp.mit.edu --recv 0x38EEACA1AB8A6E72` (this id should be equal to the last sixteen hex characters in our fingerprint).
+You can also use `gpg --fetch-key https://coreruleset.org/security.asc` directly.
+{{% /notice %}}
 
-```sh
-wget https://github.com/coreruleset/coreruleset/archive/master.zip
+The steps here assume you are using a *nix operating system. For Windows you will be doing a similar install, but probably using the zip file from our releases.
+
+To get the release file and the corresponding signature:
+
+```bash
+$ wget https://github.com/coreruleset/coreruleset/archive/refs/tags/v{{< param crs_latest_release >}}.tar.gz
+$ wget https://github.com/coreruleset/coreruleset/releases/download/v{{< param crs_latest_release >}}/coreruleset-{{< param crs_latest_release >}}.tar.gz.asc
 ```
 
-Once this is installed extract it somewhere well known on your server.
+Optional verification:
+```bash
+❯ gpg --verify coreruleset-{{< param crs_latest_release >}}.tar.gz.asc v{{< param crs_latest_release >}}.tar.gz
+gpg: Signature made Wed Jun 30 10:05:48 2021 -03
+gpg:                using RSA key 36006F0E0BA167832158821138EEACA1AB8A6E72
+gpg: Good signature from "OWASP Core Rule Set <security@coreruleset.org>" [unknown]
+gpg: WARNING: This key is not certified with a trusted signature!
+gpg:          There is no indication that the signature belongs to the owner.
+Primary key fingerprint: 3600 6F0E 0BA1 6783 2158  8211 38EE ACA1 AB8A 6E72
+```
+
+If the signature was good, the verification succeeded. If you see a warning like the above, it means you know our public key, but you are not trusting it. You can trust it by using the following method:
+
+```bash
+gpg edit-key 36006F0E0BA167832158821138EEACA1AB8A6E72
+gpg> trust
+Your decision: 5 (ultimate trust)
+Are you sure: Yes
+gpg> quit
+```
+
+Then you will see this result when verifying:
+```bash
+gpg --verify coreruleset-3.3.2.tar.gz.asc v3.3.2.tar.gz
+gpg: Signature made Wed Jun 30 15:05:48 2021 CEST
+gpg:                using RSA key 36006F0E0BA167832158821138EEACA1AB8A6E72
+gpg: Good signature from "OWASP Core Rule Set <security@coreruleset.org>" [ultimate]
+```
+
+### Installing the files
+
+Once you downloaded and verified the release, extract it somewhere well known on your server.
 Typically this will be in the webserver directory. We are demonstrating
 with Apache below. For information on configuring Nginx or IIS see
 [install]({{< ref "install.md" >}}). Additionally, while it is a
```

Comment:
```suggestion
the [extended install]({{< ref "install.md" >}}) page. Additionally, while it is a
```

There's another reference ton install.md that should be updated to "extended install" below on line 104.